addons/gloot/ctrl_inventory_grid.gd

```diff
@@ -314,18 +334,62 @@ func _select(item: InventoryItem) -> void:
     _selected_item = item
     _refresh_selection()
 
+func switch_virtual_cursor_grid(new_grid):
```

Comment:
I don't find `new_grid` very descriptive. Maybe rename it to `new_ctrl_inventory_grid` for clarity? Also, I know we can't declare it as `CtrlInventoryGrid`, but we can declare it as `Control` and that is better than leaving it as a variant, imo.